extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -73,6 +73,19 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 			resume: function () {
 				ima.getAdsManager().resume();
 			},
+			mute: function () {
+				return this.setVolume(0);
+			},
+			unmute: function () {
+				return this.setVolume(0.75);
+			},
+			volumeToggle: function () {
+				if (this.isMuted()) {
+					this.unmute();
+				} else {
+					this.mute();
+				}
```

Comment:
JS linter highlighted this structure because there is no assignment :(

[StackOverflow said the same](http://stackoverflow.com/a/2933472)